### ISSUE TITLE
Loss does't reduce. Finally appear：InvalidArgumentError (see above for traceback): Loss is inf or nan. : Tensor had NaN values

### ISSUE THREAD

**Issue by Z-Ianthe** [tensorflow/models#5939]
**What is the top-level directory of the model you are using: **models-master/research/deeplab**
Have I written custom code: **No**
OS Platform and Distribution: **Linux Ubuntu 16.04**
TensorFlow installed from:  **Anaconda**
TensorFlow version (use command below): **1.6.0**
Bazel version (if compiling from source): **N/A**
CUDA/cuDNN version: **9.0/7.0.5**
GPU model and memory: **GeForce GTX 1080 Ti****

Exact command to reproduce: 
python train.py \
  --logtostderr \
  --train_split="train" \
  --model_variant="xception_65" \
  --atrous_rates=6 \
  --atrous_rates=12 \
  --atrous_rates=18 \
  --output_stride=16 \
  --decoder_output_stride=4 \
  --train_crop_size=513 \
  --train_crop_size=513 \
  --train_batch_size=1 \
  --training_number_of_steps=1000000 \
  --fine_tune_batch_norm=False \
  --dataset="pascal_voc_seg" \
  --tf_initial_checkpoint="/root/models-master/research/deeplab/model_zoo/deeplabv3_pascal_train_aug/model.ckpt.index" \
  --train_logdir="/root/models-master/research/deeplab/deeplabv3_pascal_train_aug/model.ckpt.index" \
  `--dataset_dir="/root/models-master/research/deeplab/datasets/turbulent_voc_seg/tfrecord"

**I have tried train on voc2012 datasets with different models downloaded from model zoo ，such as
mobilenetv2_dm05_coco_voc_trainaug，mobilenetv2_coco_voc_trainaug and xception65_coco_voc_trainaug.** 

**But loss never reduce, like this:**
INFO:tensorflow:global step 27000: loss = 7.2254 (0.214 sec/step)
INFO:tensorflow:global step 27001: loss = 7.2572 (0.212 sec/step)
INFO:tensorflow:global step 27002: loss = 7.0260 (0.208 sec/step)
INFO:tensorflow:global step 27003: loss = 7.2306 (0.209 sec/step)
INFO:tensorflow:global step 27004: loss = 7.7214 (0.211 sec/step)
**Finally appear this error：**
INFO:tensorflow:global step 84940: loss = 7.2817 (0.214 sec/step)
INFO:tensorflow:global step 84950: loss = 5.1901 (0.220 sec/step)
INFO:tensorflow:global step 84960: loss = 14174.7910 (0.223 sec/step)
INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, Loss is inf or nan. : Tensor had NaN values
	 [[Node: CheckNumerics = CheckNumerics[T=DT_FLOAT, message="Loss is inf or nan.", _device="/job:localhost/replica:0/task:0/device:CPU:0"](total_loss)]]
......
Caused by op 'CheckNumerics', defined at:
  File "train.py", line 394, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/platform/app.py", line 126, in run
    _sys.exit(main(argv))
  File "train.py", line 337, in main
    total_loss = tf.check_numerics(total_loss, 'Loss is inf or nan.')
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/ops/gen_array_ops.py", line 498, in check_numerics
    "CheckNumerics", tensor=tensor, message=message, name=name)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/ops.py", line 3271, in create_op
    op_def=op_def)
  File "/usr/local/lib/python3.5/dist-packages/tensorflow/python/framework/ops.py", line 1650, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): Loss is inf or nan. : Tensor had NaN values
	 [[Node: CheckNumerics = CheckNumerics[T=DT_FLOAT, message="Loss is inf or nan.", `_device="/job:localhost/replica:0/task:0/device:CPU:0"](total_loss)]]

**And if training_number_of_steps is smaller，no nan value error. But `miou_1.0[0.0349133424]`**

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by Z-Ianthe** [comment]
Solved it. I made a stupid mistake. 
The model should import model.ckpt instead of model.ckpt.index.
